tools/tcat_ble_client/cli/base_commands.py

```diff
@@ -103,6 +105,38 @@ async def execute_default(self, args, context):
         return CommandResultTLV(tlv_response)
 
 
+class PingCommand(Command):
+
+    def get_help_string(self) -> str:
+        return 'Echo request to device.'
+
+    async def execute_default(self, args, context):
+        bless: BleStreamSecure = context['ble_sstream']
+        size = 10
+        max_payload = 512
+        if len(args) > 0:
+            size = int(args[0])
+            if size > max_payload:
+                print(f'Payload to large. Maximum supported value is {max_payload}')
+                return
+        to_send = token_bytes(size)
+        data = TLV(TcatTLVType.PING.value, to_send).to_bytes_extended()
+        elapsed_time = time()
+        response = await bless.send_with_resp(data)
+        print(response)
+        elapsed_time = time() - elapsed_time
```

Comment:
```suggestion
        elapsed_time = time() - elapsed_time
        print(response)
```
to measure only sending process, without `print()` time

Reply:
fixed.